### PR TITLE
Feat/responsive how

### PR DESCRIPTION
**Pull request by rileydanejohnston**
features:
- moved the steps to their own component
- moved step data to const file
- refactored how/step components to be reusable, the differences come from props now
- implemented responsive design

I know I said I wasn't going to totally refactor everything but I had a breakthrough. I think this makes the markup a lot cleaner, which is good. But I also think it takes a lot of CSS and puts it in the const file, which I'm not crazy about. Let me know what you think.

### CODE REVIEW

File: src/components/how/styledHow.js [hunk 1]
```diff
@@ -24,11 +31,27 @@ export const HowTitle = styled.h2`
   line-height: 52px;
   letter-spacing: 0em;
   margin-bottom: 42px;
+
+  @media( max-width: 1023px) {
+    text-align: center;
+  }
+
+  @media( max-width: 767px) {
```
Comment (trisboyd) [inline src/components/how/styledHow.js:39]: This is just a preference thing, but I think the font-size drops too much at 767.  I would put an intermediate size in there until 425.

File: src/constants/howConst.js [hunk 2]
```diff
@@ -6,4 +14,183 @@ export const HowData = {
   step4: `04.\nWash containers\nat commercial facility`,
   highlighter: 'Circular ecosystem',
   buttonText: 'Work with Us'
+};
+
+export const HowInfo = {
```
Comment (trisboyd) [inline src/constants/howConst.js:9]: This is awesome.  Great work refactoring this, it looks like it was a huge pain, but it turned out great.

File: src/constants/howConst.js [hunk 3]
```diff
@@ -1,3 +1,11 @@
+import phoneImg from '../images/phone.svg';
+import containerImg from '../images/container-monster.svg';
+import truckImg from '../images/truck.svg';
+import starsImg from '../images/stars.svg';
+import curveArrow from '../images/up-right-arrow.svg';
+import straightArrow from '../images/straight-arrow.svg';
+import mobile180Arrow from '../images/mobile-u-arrow.svg';
+
 export const HowData = {
   title: 'How it works?',
```
Comment (trisboyd) [inline src/constants/howConst.js]: This just occurred to me: 'how it works?' is not good grammar.  I feel that there shouldn't be a question mark there.  I know the design has one, but it doesn't make any sense

**Review by trisboyd — APPROVED** [review]
Really nice job.  I have a couple suggestions but on the whole this was a huge task and you did great.

File: src/constants/howConst.js (hunk 3 above)
Comment (rileydanejohnston) [reply src/constants/howConst.js]: I agree, I will change it

File: src/components/how/styledHow.js (hunk 1 above)
Comment (rileydanejohnston) [reply src/components/how/styledHow.js:39]: I made it so at 767 it jumps down to 36px then at 425 down to 28px. I think it looks better now. Good call

**Pull request merged**